omaha/client/client.go

```diff
@@ -222,36 +245,52 @@ func (ac *AppClient) Ping() error {
 		return err
 	}
 
-	if appResp.Ping == nil {
+	// BUG: CoreUpdate does not send ping status in response.
+	_ = appResp
+	/*if appResp.Ping == nil {
 		return fmt.Errorf("omaha: ping status missing from response")
 	}
 
 	if appResp.Ping.Status != "ok" {
 		return fmt.Errorf("omaha: ping status %s", appResp.Ping.Status)
-	}
+	}*/
 
 	return nil
 }
 
-func (ac *AppClient) Event(event *omaha.EventRequest) error {
+// Event asynchronously sends the given omaha event.
+// Reading the error channel is optional.
+func (ac *AppClient) Event(event *omaha.EventRequest) <-chan error {
+	errc := make(chan error, 1)
+	url := ac.apiEndpoint
 	req := ac.newReq()
 	app := req.Apps[0]
 	app.Events = append(app.Events, event)
 
-	appResp, err := ac.doReq(ac.apiEndpoint, req)
-	if err != nil {
-		return err
-	}
-
-	if len(appResp.Events) == 0 {
-		return fmt.Errorf("omaha: event status missing from response")
-	}
-
-	if appResp.Events[0].Status != "ok" {
-		return fmt.Errorf("omaha: event status %s", appResp.Events[0].Status)
-	}
-
-	return nil
+	go func() {
+		appResp, err := ac.doReq(url, req)
+		if err != nil {
+			errc <- err
+			return
+		}
+
+		// BUG: CoreUpdate does not send event status in response.
```

Comment:
nit: Maybe link the issue in the comments?
https://github.com/coreos-inc/roller/issues/750